docs/standard/native-interop/exceptions-interoperability.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Exceptions Interoperability"
+ms.date: "01/16/2020"
+ms.technology: dotnet-standard
+helpviewer_keywords:
+  - "unmanaged code, exceptions"
+  - "exceptions, unmanaged code"
+  - "interop, exceptions"
+  - "exceptions, interop"
+---
+# Working with Interop Exceptions in Unmanaged Code
+Unmanaged code exception interop is not supported in non-Windows platforms, due to the portability issues that arise, since managed code can't know how these exception mechanisms work under the covers. The Unix ABI has no definition for this and therefore, exceptions end up resulting in unpredictable behaviors and potential crashes.
+
+## Setjmp/Longjmp Behaviors
+`Setjmp`/`Longjmp` interop is held to the same constraints as exception handling as it is not supported to skip over managed frames this way.
```

Comment:
```suggestion
The interop with `setjmp`/`longjmp` C functions is not supported. It is not supported to use `longjmp` to skip over managed frames.
```

Reply:
Could you please apply this suggestion as well?

Reply:
Sorry Jan, for some reason, it didn't show up in the email and I overlooked it when reviewing here. Let me do it.